sensing/autoware_cuda_pointcloud_preprocessor/src/cuda_pointcloud_preprocessor/cuda_pointcloud_preprocessor.cu

```diff
@@ -37,68 +40,44 @@ namespace autoware::cuda_pointcloud_preprocessor
 
 CudaPointcloudPreprocessor::CudaPointcloudPreprocessor()
 {
-  sensor_msgs::msg::PointField x_field, y_field, z_field, intensity_field, return_type_field,
-    channel_field, azimuth_field, elevation_field, distance_field, time_stamp_field;
-  x_field.name = "x";
-  x_field.offset = 0;
-  x_field.datatype = sensor_msgs::msg::PointField::FLOAT32;
-  x_field.count = 1;
-
-  y_field.name = "y";
-  y_field.offset = 4;
-  y_field.datatype = sensor_msgs::msg::PointField::FLOAT32;
-  y_field.count = 1;
-
-  z_field.name = "z";
-  z_field.offset = 8;
-  z_field.datatype = sensor_msgs::msg::PointField::FLOAT32;
-  z_field.count = 1;
-
-  intensity_field.name = "intensity";
-  intensity_field.offset = 12;
-  intensity_field.datatype = sensor_msgs::msg::PointField::UINT8;
-  intensity_field.count = 1;
-
-  return_type_field.name = "return_type";
-  return_type_field.offset = 13;
-  return_type_field.datatype = sensor_msgs::msg::PointField::UINT8;
-  return_type_field.count = 1;
-
-  channel_field.name = "channel";
-  channel_field.offset = 14;
-  channel_field.datatype = sensor_msgs::msg::PointField::UINT16;
-  channel_field.count = 1;
-
-  static_assert(sizeof(OutputPointType) == 16, "OutputPointType size is not 16 bytes");
-  static_assert(offsetof(OutputPointType, x) == 0);
-  static_assert(offsetof(OutputPointType, y) == 4);
-  static_assert(offsetof(OutputPointType, z) == 8);
-  static_assert(offsetof(OutputPointType, intensity) == 12);
-  static_assert(offsetof(OutputPointType, return_type) == 13);
-  static_assert(offsetof(OutputPointType, channel) == 14);
```

Comment:
These `static_assert` seem to be dropped. Is it intended?

Reply:
Added in 
5a97d81